Engine/AI/NavMesh.cpp

```diff
@@ -477,7 +475,7 @@ void NavMesh::InitAABB()
 	}
 }
 
-bool NavMesh::FindPath(float3& start, float3& end, std::vector<float3>& path)
+bool NavMesh::FindPath(float3& start, float3& end, std::vector<float3>& path, PathMode path_mode)
```

Comment:
path_mode is used?

Reply:
That method is for pathfinding, that have different modes (and will have more), for vs1 is not used but it will be used in the future